blog/authors.yml

```diff
@@ -17,7 +17,7 @@ fzhou:
 
 yzha:
   name: Yi Zha
-  title:  Engineer, Microsoft
+  title:  ORAS Maintainer
```

Comment:
Currently, there is no public record that Yi is a maintainer of any repo in `oras-project`.

Reply:
@shizhMSFT Thanks. I updated Yi's title.